Room/Room.cpp

```diff
@@ -74,7 +74,7 @@ Room::Room() : _inputHandler(&Room::_manageInterprocessCommunication, this)
     _name = std::string(10, '\0');
 }
 
-Room::Room(unsigned short id, std::string name, Client networkInformations)
+Room::Room(unsigned short id, std::string name, Client networkInformations, short *configs)
```

Comment:
👀

Reply:
configurations are used to add modifier on room creation. There are raw pointer because they are used by the network.